### PR TITLE
add language at optional configurations

### PR DESCRIPTION
**Pull request by wataru420**
I want to use in Japanese because I'm Japanese.
So I added language option in configuration.

Please check it.

### CODE REVIEW

**Comment by jvarness** [comment]
@wataru420 thank you for your PR! I will review this change later when I arrive at work.

File: README.md [hunk 1]
```diff
@@ -41,10 +41,12 @@ The following configurations are optional:
 ```coffeescript
 HUBOT_YELP_DEFAULT_LOCATION   # The default location that should be used
 HUBOT_YELP_DEFAULT_CATEGORY   # The default category that should be used
+HUBOT_YELP_DEFAULT_LANG       # The default language that should be used
 ```
 
 If a location is not specified, the script is hard-coded to use Kansas City, MO as the default location. If no 
-default category is specified, no category will be used unless specified by the user.
+default category is specified, no category will be used unless specified by the user.If a language is not specified,
```
Comment (jvarness) [inline README.md]: I'm being nit-picky about this, but can you put a space in between the period and "If"?

Comment (wataru420) [inline README.md]: I improved that line that was pointed out. Please check it!

**Comment by jvarness** [comment]
@wataru420 looks good! I'll run a regression test later on this pull request and if I like the change I'll make a release! I'll notify you when it's released.

**Comment by wataru420** [comment]
@jvarness Thank you in advance!

**Comment by jvarness** [comment]
@wataru420 I think I might actually enhance the code to include your changes as well as add a command for the user to specify the locale.

**Comment by wataru420** [comment]
@jvarness Thanks to merged it. I was also thinking about that, but I think there is no reason to changing the language in the middle. How do you think?

**Comment by jvarness** [comment]
I thought about it some more, and I think it's probably fine to leave it the way that it is now.

**Comment by jvarness** [comment]
@wataru420 not sure if you saw but this has been released with 0.1.3.

Thank you for your contribution!

**Comment by wataru420** [comment]
Thank you! :smile:

**Pull request merged**